### ISSUE TITLE
 Undefined index: vcard in TelegramContactDriver

### ISSUE THREAD

**Issue by vdomah** [botman/driver-telegram#82]
### Required Information
- BotMan version: 2.6
- BotMan Telegram Driver version: 1.6

### Expected behaviour
Receiving contact array after user sends his contact details.

### Actual behaviour
TelegramContactDriver is not checking iv vcard field is present. That causes error:
 ````
Undefined index: vcard {"exception":"[object] (ErrorException(code: 0): Undefined index: vcard at /var/www/html/vendor/botman/driver-telegram/src/TelegramContactDriver.php:55)
````

### Steps to reproduce
Using this code example: https://gist.github.com/yarkm13/4d9359f2c96149a012f563ea41849c02

### Extra details
Solved it by extending TelegramContactDriver and overriding loadMessages method:
````
public function loadMessages()
    {
        $message = new IncomingMessage(Contact::PATTERN, $this->event->get('from')['id'], $this->event->get('chat')['id'], $this->event);
        $message->setContact(new Contact(
            $this->event->get('contact')['phone_number'],
            $this->event->get('contact')['first_name'],
            $this->event->get('contact')['last_name'],
            $this->event->get('contact')['user_id'],
            $this->event->get('contact')['vcard'] ?? null
        ));

        $this->messages = [$message];
    }
````
then in routes/botman.php:
````
DriverManager::unloadDriver(TelegramContactDriverBase::class);
DriverManager::loadDriver(TelegramContactDriver::class);
// required to do before the 'resolve'
$botman = resolve('botman');
````

**Comment by pedroufv** [comment]
same problem here...the bot keep stop when don't have vcard

**Comment by EduardMalik** [comment]
Same problem too. 

`POST   -d '{"update_id":513976669,
"message":{"message_id":200,"from":{"id":1218871752,"is_bot":false,"first_name":"Eduard","last_name":"Malik","username":"eduard_malik","language_code":"ru"},"chat":{"id":1218871752,"first_name":"Eduard","last_name":"Malik","username":"eduard_malik","type":"private"},"date":1600069362,"contact":{"phone_number":"79789999999","first_name":"Eduard","last_name":"Malik","user_id":1218871752}}}'`

`ErrorException: Undefined index: vcard in file /home/hotadd/www/chatbot/vendor/botman/driver-telegram/src/TelegramContactDriver.php on line 55
Stack trace:
  1. ErrorException-&gt;() /home/hotadd/www/chatbot/vendor/botman/driver-telegram/src/TelegramContactDriver.php:55
  2. Illuminate\Foundation\Bootstrap\HandleExceptions-&gt;handleError() /home/hotadd/www/chatbot/vendor/botman/driver-telegram/src/TelegramContactDriver.php:55
  3. BotMan\Drivers\Telegram\TelegramContactDriver-&gt;loadMessages() /home/hotadd/www/chatbot/vendor/botman/driver-telegram/src/TelegramContactDriver.php:38
  4. BotMan\Drivers\Telegram\TelegramContactDriver-&gt;getMessages() /home/hotadd/www/chatbot/vendor/botman/botman/src/BotMan.php:192
  5. BotMan\BotMan\BotMan-&gt;getMessages() /home/hotadd/www/chatbot/vendor/botman/botman/src/Traits/HandlesConversations.php:170
  6. BotMan\BotMan\BotMan-&gt;loadActiveConversation() /home/hotadd/www/chatbot/vendor/botman/botman/src/BotMan.php:422
  7. BotMan\BotMan\BotMan-&gt;listen() /home/hotadd/www/chatbot/routes/botman.php:26
  8. Illuminate\Routing\RouteFileRegistrar-&gt;{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Routing/Route.php:205
  9. Illuminate\Routing\Route-&gt;runCallable() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Routing/Route.php:179
 10. Illuminate\Routing\Route-&gt;run() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Routing/Router.php:681
 11. Illuminate\Routing\Router-&gt;Illuminate\Routing\{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:130
 12. Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php:41
 13. Illuminate\Routing\Middleware\SubstituteBindings-&gt;handle() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:171
 14. Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php:77
 15. Illuminate\Foundation\Http\Middleware\VerifyCsrfToken-&gt;handle() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:171
 16. Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php:49
 17. Illuminate\View\Middleware\ShareErrorsFromSession-&gt;handle() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:171
 18. Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php:56
 19. Illuminate\Session\Middleware\StartSession-&gt;handle() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:171
 20. Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php:37
 21. Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse-&gt;handle() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:171
 22. Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php:67
 23. Illuminate\Cookie\Middleware\EncryptCookies-&gt;handle() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:171
 24. Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:105
 25. Illuminate\Pipeline\Pipeline-&gt;then() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Routing/Router.php:683
 26. Illuminate\Routing\Router-&gt;runRouteWithinStack() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Routing/Router.php:658
 27. Illuminate\Routing\Router-&gt;runRoute() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Routing/Router.php:624
 28. Illuminate\Routing\Router-&gt;dispatchToRoute() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Routing/Router.php:613
 29. Illuminate\Routing\Router-&gt;dispatch() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:170
 30. Illuminate\Foundation\Http\Kernel-&gt;Illuminate\Foundation\Http\{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:130
 31. Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}() /home/hotadd/www/chatbot/vendor/fideloper/proxy/src/TrustProxies.php:57
 32. Fideloper\Proxy\TrustProxies-&gt;handle() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:171
 33. Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php:21
 34. Illuminate\Foundation\Http\Middleware\TransformsRequest-&gt;handle() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:171
 35. Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php:21
 36. Illuminate\Foundation\Http\Middleware\TransformsRequest-&gt;handle() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:171
 37. Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php:27
 38. Illuminate\Foundation\Http\Middleware\ValidatePostSize-&gt;handle() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:171
 39. Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php:63
 40. Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode-&gt;handle() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:171
 41. Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:105
 42. Illuminate\Pipeline\Pipeline-&gt;then() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:145
 43. Illuminate\Foundation\Http\Kernel-&gt;sendRequestThroughRouter() /home/hotadd/www/chatbot/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:110
 44. Illuminate\Foundation\Http\Kernel-&gt;handle() /home/hotadd/www/chatbot/public/index.php:55`

**Comment by crynobone** [comment]
Fixed via botman/driver-telegram#88 and will be released as `1.6.2` later today.
